tests/components/tessie/test_coordinator.py

```diff
@@ -27,81 +28,84 @@
 WAIT = timedelta(seconds=TESSIE_SYNC_INTERVAL)
 
 
-async def test_coordinator_online(
-    hass: HomeAssistant, mock_get_state, mock_get_status, freezer: FrozenDateTimeFactory
-) -> None:
-    """Tests that the coordinator handles online vehicles."""
+async def test_coordinator(hass: HomeAssistant, freezer: FrozenDateTimeFactory) -> None:
+    """Tests that the coordinator updates vehicles."""
 
-    await setup_platform(hass, PLATFORMS)
+    await setup_platform(hass, [Platform.BINARY_SENSOR])
 
     freezer.tick(WAIT)
     async_fire_time_changed(hass)
     await hass.async_block_till_done()
-    mock_get_status.assert_called_once()
-    mock_get_state.assert_called_once()
-    assert hass.states.get("binary_sensor.test_status").state == STATE_ON
+
+    # Since the fixture data is old, the status will be offline
+    assert hass.states.get("binary_sensor.test_status").state == STATE_OFF
 
 
-async def test_coordinator_asleep(
-    hass: HomeAssistant, mock_get_status, freezer: FrozenDateTimeFactory
+async def test_coordinator_online(
```

Comment:
We could just combine this test with the previous as you start with old data and once asserting it's sleeping you can update the timestamp to make it alive again.

Very easy to follow as you write the short comments in the tests 👍